pkg/graph/schema.graphql

```diff
@@ -738,6 +762,7 @@ type Query {
     after: String
     first: Int!
   ): AccessibilityRequestsConnection
+  grtFeedbacks(intakeID: UUID!): [GRTFeedback!] @hasRole(role: EASI_GOVTEAM)
```

Comment:
Why not embed this under the `SystemIntake` so that it's possible to get them along with the system intake if we wanted to?

Reply:
I think we might need both? The business owner isn't seeing the system intake from the graph endpoint yet.

Reply:
How about this: for now, I'll leave it how it is. Once biz case is covered under gql, we can make the switch.